colossalai/kernel/triton/int8_rotary_embedding_kernel.py

```diff
@@ -0,0 +1,117 @@
+# Adapted from ModelTC https://github.com/ModelTC/lightllm
```

Comment:
Same problem. Can we import this lib directly?

Reply:
The method support int8 rortary embedding. The method in light lllm can't support.